content/en/api/service_level_objectives/service_level_objectives_delete.md

```diff
@@ -11,4 +11,6 @@ Permanently delete a SLO.
 
 **ARGUMENTS**:
 
-This endpoint takes no JSON arguments.
+* **`force`** [*optional*]:
+
+    Boolean: Force delete SLO.  Required if SLO exists in a dashboard.  Default `false`
```

Comment:
Could you reword this as: 

> Permanently delete an SLO. 
>
>  If an SLO is used in a dashboard, the `DELETE /v1/slo/` endpoint will return a `409` conflict error because the SLO is referenced in a dashboard.
>
> **ARGUMENTS**:
>
> * **`force`** [*optional*]:
>
> Boolean: Force delete the SLO. The SLO will be deleted regardless of if it's referenced in a dashboard.  Default`false`.

Reply:
@armcburney nice additions.  Just added 👍